src/fal/fal_script.py

```diff
@@ -64,12 +64,20 @@ class FalScript:
     model: Optional[DbtModel]
     path: Path
     _faldbt: FalDbt
-
-    def __init__(self, faldbt: FalDbt, model: Optional[DbtModel], path: str):
+    _is_post_hook: bool
+
+    def __init__(
+        self,
+        faldbt: FalDbt,
+        model: Optional[DbtModel],
+        path: str,
+        is_post_hook: bool = False,
```

Comment:
Add a todo here that we wont need this property when after scripts are deprecated